src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -70,6 +70,7 @@
         ("swinv2", "ViTFeatureExtractor"),
         ("van", "ConvNextFeatureExtractor"),
         ("videomae", "VideoMAEFeatureExtractor"),
+        ("vision-encoder-decoder", "ViTFeatureExtractor"),
```

Comment:
@Narsil,

I am not sure about this. There is nothing that "forces" future vision-encoder-decoder to use VIT as an encoder. But if we remove it, we loses support for https://huggingface.co/ydshieh/vit-gpt2-coco-en-ckpts